packages/core/package.json

```diff
@@ -37,7 +37,6 @@
     "babel-polyfill": "6.26.0",
     "copy-webpack-plugin": "4.5.2",
     "css-loader": "0.28.7",
-    "d3": "4.11.0",
```

Comment:
Don't we still need a copy of `d3` in our devDependencies?

Reply:
We should also probably look at removing the copy from `dependencies` since we have it listed in `peerDependencies` ... 

Or remove it from `peerDependencies`, but I don't think that's super viable

Reply:
I'm still not sure about what we should do with `d3`...

We can do peer and dev, but that'd make sense if `d3` doesn't end up in our bundle...

Reply:
@zvonimirfras

Reply:
peer and dev is generally what we do for shared dependencies (angular, react, etc) ... I think we should bring this up in the slack channel - see what everyone thinks, and make sure they're aware we're (probably) changing to not install it by default

Reply:
peer + dev makes sense
npm/yarn should warn that there's a not-installed peer dependency and additionally informing people of this change is useful and appreciated. 👍

Reply:
You guys are right. I wasn't thinking clearly about dev dependencies